Decred Wallet.xcodeproj/xcshareddata/xcschemes/Decred Wallet.xcscheme

```diff
@@ -28,6 +28,16 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
+         <TestableReference
```

Comment:
Related to the issue resolved by this PR? If not, please remove.

Reply:
resolved in 4c21651